public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -370,7 +317,7 @@ mciModule.controller('PerformanceDiscoveryCtrl', function (
         enableFiltering: false,
       },
       {
-        name: 'ops/sec',
+        name: vm.metric_name ? vm.metric_name : 'ops/sec',
```

Comment:
What is the difference between 'ops/sec' and 'ops_per_sec'?

Maybe 'ops/sec' is just the column name rather than the field name.

Reply:
Correct, ops/sec is the default column name whereas the field name is ops_per_sec. This does bring up an inconsistency since you can select the ops_per_sec metric which will change the column label to that, but I think that's not a big deal

Reply:
:+1: